### PR TITLE
apiservers: added unit tests for openshiftapiserver.go

### PR DESCRIPTION
**Pull request by kononovn**

### CODE REVIEW

File: pkg/apiservers/openshiftapiserver_test.go [hunk 1]
```diff
@@ -0,0 +1,290 @@
...
+
+		if testCase.expectedError == nil {
+			assert.Equal(t, testOpenshiftAPIServer.Name, testCase.testOpenshiftAPIServerBuilder.Definition.Name)
+		} else {
+			assert.Equal(t, err.Error(), testCase.expectedError.Error())
+		}
+	}
+}
+
+func TestOpenshiftAPIServerExist(t *testing.T) {
```
Comment (sebrandon1) [inline pkg/apiservers/openshiftapiserver_test.go:100]:
```suggestion
func TestOpenshiftAPIServerExists(t *testing.T) {
```

**Review by sebrandon1 — APPROVED** [review]
One comment but lgtm

File: pkg/apiservers/openshiftapiserver_test.go [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+			assert.Equal(t, "", msg)
+		}
+	}
+}
+
+func TestOpenshiftAPIServerWaitUntilConditionTrue(t *testing.T) {
+	testCases := []struct {
+		testOpenshiftAPIServerBuilder *OpenshiftAPIServerBuilder
+		condition                     string
+		conditionStatus               operatorv1.ConditionStatus
```
Comment (achuzhoy) [inline pkg/apiservers/openshiftapiserver_test.go:174]: redundant?

**Comment by achuzhoy** [comment]
One non deal breaking comment and lgtm overall

**Pull request merged**